vunit.dockerfile

```diff
@@ -1,28 +1,49 @@
 ARG TAG="buster-mcode"
 
+#---
+
 FROM ghdl/ghdl:$TAG AS base
+
 ARG PY_PACKAGES
-RUN apt-get update -qq \
+
+RUN set -ux \
+ && apt-get update -qq \
  && DEBIAN_FRONTEND=noninteractive apt-get -y install --no-install-recommends \
```

Comment:
These images are used not only with VUnit, but also for any project using GHDL together with Python. For co-simulation, users of ghdl-cosim or cocotb can use these images. That's why I'm hesitant to remove `make`. Which, otherwise, I believe is included in the base images anyway.

Regarding `curl`, since `git` is not included in this images, it's provided as a mechanism to retrieve and extract git repos and other dependencies. It takes ~12 MB (uncompressed), so I'd cosider it to be negligible. I'd keep it. `ca-certificates` is a dependency of `curl`, so we might remove it.

Reply:
Okay, got it. I will reinstate `make` and `curl`.